MiniApp/Classes/MiniApp.swift

```diff
@@ -15,11 +15,12 @@ public class MiniApp: NSObject {
     /// Create a Mini App for the given appId. Mini app will be downloaded and cached in local.
```

Comment:
still says "for the given appId"

Reply:
Noted. Will push the changes in the next PR